tests/datacache/test_datacache.py

```diff
@@ -34,3 +36,50 @@ def test_cache(tmp_path: "Path") -> None:
     assert len(cache) == 1
     cache.clear()
     assert len(cache) == 0
+
+
+def test_numpy(tmp_path: "Path") -> None:
+    """Test adding numpy arrays to the datachache."""
+    try:
+        import numpy as np
+    except ImportError:
+        pytest.skip("numpy is not installed")
+        return
+
+    from oteapi.datacache import DataCache
+
+    cache = DataCache(cache_dir=tmp_path / "oteapi-test_numpy")
+    val = np.eye(4)
+    key = cache.add(val)
+
+    assert np.all(cache[key] == np.eye(4))
+    cache.clear()
+
+
+def test_ase_atoms(tmp_path: "Path") -> None:
+    """Test adding ase Atoms objects to the datachache."""
+    try:
+        import ase
+    except ImportError:
+        pytest.skip("ase is not installed")
+        return
+
+    from ase.io.jsonio import MyEncoder
```

Comment:
Nice, how did yoy find this? It is not documented as fas as I can see.
https://wiki.fysik.dtu.dk/ase/ase/io/io.html?highlight=io#module-ase.io